Dockerfile

```diff
@@ -4,11 +4,20 @@ LABEL description="Image with k8s-handle" \
       maintainer="<Infrastructure & Operations> io@2gis.ru" \
       source="https://github.com/rvadim/k8s-handle"
 
+RUN apk add --no-cache \
```

Comment:
What reason for this change? Bug, simplicity, performance?

Reply:
re-build of image is faster, because requirements installed in separate layer, just improvement